envs.sh.sample

```diff
@@ -4,6 +4,7 @@
 
 export AWS_PROFILE=undefined
 export AWS_REGION=us-west-2
+# export AWS_ROLE_NAME= optional role name if using sts roles
```

Comment:
When this variable is defined it will be used to create an `AWS_ROLE_ARN` variable.  That will intern be used to populate the `provider.tf` generated file with proper role based config.